src/build.rs

```diff
@@ -446,11 +446,26 @@ impl BuildQueue {
             let mut manifest_path = build_dir.clone();
             manifest_path.push("Cargo.toml");
             trace!("manifest_path: {:?}", manifest_path);
+            // TODO: Add support for virtual manifests and multiple packages 
             let ws = Workspace::new(&manifest_path, &config).expect("could not create cargo workspace");
+            let current_package = ws.current().unwrap(); 
+            let targets = current_package.targets(); 
+
+            let bins = {
```

Comment:
Declared like this, at this scope, because `CompileFilter` accepts a slice of names that has to live at least as long as `CompileOptions`, `Workspace`